### ISSUE TITLE
`CTRL` + `Left Click` shortcut for "open new tab" in File Manager

### ISSUE THREAD

**Issue by IgnacioHeredia** [Zettlr/Zettlr#3387]
### Description

As the title says it would be nice to have a shortcut (`CTRL` + `Left Click`) to open a file in a **new** tab when clicking on it in the File Manager. 

I'm aware you could unmark the option `Avoid opening files in new tabs if possible` (side note: the phrasing of the option seems odd 🙃️). But I would like to have this behaviour **selectively**. That is (1) if clicking normally then replace the current tab, (2) if clicking with `CRTL` pressed then open in new tab. 

I think this would improve UX as it mimics more closely the default behaviour of a web browser.

Many thanks!

Ignacio

### Proposed Changes

Add shortcut (`CTRL` + `Left Click`) to open a file in a **new** tab when clicking on it in the File Manager. 

### Caveats

None that I can see.

### Do you Wish to Attempt Implementing this Yourself?

No

### Zettlr Version

Stable (most recent version)

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Operating System Version

Ubuntu 20.04

**Comment by leo-hstone** [comment]
Middle moude button already works!

**Comment by nathanlesage** [comment]
You are right in that a middle mouse button is not an option if you're dealing with a trackpad which obviously doesn't have a middle mouse button. HOWEVER, Ctrl+Click is already reserved for the context menu (and that is universal, iirc). Therefore, what we could do would simply be an Alt-Click? (I just tested it, and on Chrome it's a shortcut for "Save target as" :D)

**Comment by IgnacioHeredia** [comment]
> Middle mouse button already opens files in new tabs smiley This also mimics the behaviour of a web browser

Wasn't aware of this shortcut even for browsers, you just made my day 😄 

> what we could do would simply be an Alt-Click

Maybe not as intuitive but sounds like a nice compromise!
